data/moves.ts

```diff
@@ -4671,6 +4756,32 @@ export const BattleMovedex: {[moveid: string]: MoveData} = {
 		target: "normal",
 		type: "Dragon",
 	},
+	expandingforce: {
+		num: 797,
+		accuracy: 100,
+		basePower: 80,
+		category: "Special",
+		desc: "If the current active terrain is Psychic Terrain, this move's base power is boosted, and this move hits all opposing Pokemon.",
+		shortDesc: "Psychic Terrain:Boosted power+hits all foes.",
+		name: "Expanding Force",
+		pp: 20,
+		priority: 0,
+		flags: {protect: 1, mirror: 1},
+		onBasePower(basePower) {
+			if (this.field.getTerrain().id === 'psychicterrain') {
```

Comment:
```suggestion
			if (this.field.isTerrain('psychicterrain')) {
```

(similarly in other places where you check for a single terrain)